src/pybind/mgr/influx/module.py

```diff
@@ -25,6 +25,9 @@ def __init__(self, *args, **kwargs):
         self.event = Event()
         self.run = True
 
+    def get_fsid(self):
+        return self.get('osd_map')['fsid']
```

Comment:
I'd suggest using monmap here, just to save a few cycles as osdmap can be rather large.